### PR TITLE
Added my CSS animation

### PR DESCRIPTION
**Pull request by AryanCoder2010**
*Added css animation

### CODE REVIEW

File: include.js [hunk 1]
```diff
@@ -2942,6 +2942,12 @@ let cards = [
     author: "afonsocarlos",
     githubLink: "https://github.com/afonsocarlos",
   },
+  {
```
Comment (donmarvex) [inline include.js:2945]: Hello @AryanCoder2010 , please add the **"artName"** property.

**Review by donmarvex — CHANGES_REQUESTED** [review]
Hello @AryanCoder2010 , you forgot to add your artname property. Please do so and commit the changes.

File: include.js (hunk 1 above)
Comment (AryanCoder2010) [reply include.js:2945]: Thanks @donmarvex , I have created a new pull request with "artName" property.

Comment (donmarvex) [reply include.js:2945]: You can just commit the changes directly to this same branch. No need for another pull request 👍

Comment (donmarvex) [reply include.js:2945]: Looks cool. Great work.
Please review @sophiabrandt

**Comment by AryanCoder2010** [comment]
Kindly approve my changes.

**Comment by AryanCoder2010** [comment]
Kindly approve my changes.

**Review by donmarvex — APPROVED** [review]
All good here. Can successfully merge @sophiabrandt

**Pull request merged**